pkg/clustermesh/types/option.go

```diff
@@ -135,3 +135,33 @@ func (_ QuirksConfig) Flags(flags *pflag.FlagSet) {
 			"policy both within and across clusters")
 	flags.MarkHidden("allow-unsafe-policy-skb-usage")
 }
+
+const PolicyAnyCluster = ""
+
+// PolicyConfig allows the user to configure config related to ClusterMesh and policies
+type PolicyConfig struct {
+	// PolicyDefaultLocalCluster control whether policy rules assume
+	// by default the local cluster if not explicitly selected
+	PolicyDefaultLocalCluster bool
+}
+
+var DefaultPolicyConfig = PolicyConfig{
+	PolicyDefaultLocalCluster: false,
+}
+
+func (_ PolicyConfig) Flags(flags *pflag.FlagSet) {
+	flags.Bool(
+		"policy-default-local-cluster", false,
+		"Control whether policy rules assume by default the local cluster if not explicitly selected",
+	)
+}
```

Comment:
Maybe I wonder if these definitions should be directly moved to the policy package, given that not strictly clustermesh related.

Reply:
Since ClusterInfo is defined there and that it works mostly with that (except if done through the daemon legacy config :sweat_smile:), I figured it could be a good enough place but if anyone has a better idea I am happy to move this elsewhere ofc!